### ISSUE TITLE
Feature: Radios similar to what's found in Portal (updateitem1 and updateitem2)

### ISSUE THREAD

**Issue by LemonLeezard** [ChaosInitiative/Portal-2-Community-Edition#540]
## Which component should be improved?
Entity

## Describe your suggestion
I always thought that the radios that were added to the original Portal for the ARG Valve did could be a neat tool for map makers. The entities used to make them work were "updateitem1", which was a location marker to trigger "updateitem2", the entity with the radio model and through which it would play the various SSTV and Morse code sound files via proximity to updateitem1.

Being that currently, those two entities don't work correctly when trying to place them into a map (either view Hammer or in-game), having properly working versions of these two entities in Hammer could be a neat addition for map makers.

## Expected result
For the location marker, "updateitem1" could instead be named "point_updateitem_marker", and could include options to set a certain sound file for when you get close enough to the entity, as well as possibly a way to adjust the range/proximity the player needs to be at to trigger the sounds to play through the radio.

Speaking of the radio, "updateitem2" could be "point_updateitem_detector", and would obviously have similar parameters to a physics prop, though perhaps with the options to change the model used for the radio, and perhaps the option to change the default music that plays when the radio isn't near a marker.

**Comment by TeamSpen210** [comment]
I'd suggest `info_portal_radio_target`, and `prop_portal_radio` respectively. That would make the names actually describe what they do.

**Comment by KaziPlays** [comment]
I think there should be a skin selector, for selecting the Dirty and Clean skins, and a sound selector, for the music.

**Comment by Luke18033** [comment]
No, there will be a model selector which lets you select any model and skin, whether it's clean, dirty, or something else. Hardcoding model/skin options is one of the very things P2CE is trying to avoid, so please stop suggesting things be implemented that way.
